README.md

```diff
@@ -275,39 +259,32 @@ Specifies the input file for the MAGPIE execution. This can be a regular file or
 
 Specifies the output file for the MAGPIE execution. Warnings and errors still get printed to `stderr`.
 
-#### console
-
-Specifies that MAGPIE should run in console mode.
+#### mode
 
-#### ucgi
-
-Specifies that MAGPIE should run in ucgi mode.
+Specifies that MAGPIE should run in console or UCGI mode.
 
 ## Examples
 
 Run a Montecarlo simulation as a one-off:
 
 ```
-./bin/magpie go sim threads 7 numplays 5 cond 99 i 10000 info 100 check 500 plies 5 cgp C14/O2TOY9/mIRADOR8/F4DAB2PUGH1/I5GOOEY3V/T4XI2MALTHA/14N/6GUM3OWN/7PEW2DOE/9EF1DOR/2KUNA1J1BEVELS/3TURRETs2S2/7A4T2/7N7/7S7 EEEIILZ/ 336/298 0 lex NWL20;
+cgp C14/O2TOY9/mIRADOR8/F4DAB2PUGH1/I5GOOEY3V/T4XI2MALTHA/14N/6GUM3OWN/7PEW2DOE/9EF1DOR/2KUNA1J1BEVELS/3TURRETs2S2/7A4T2/7N7/7S7 EEEIILZ/ 336/298 0 -lex NWL20
 ```
 
-Run a static simulation in console mode:
-
 ```
- ./bin/magpie console
-magpie>go sim static plies 1 threads 1 numplays 15 cgp C14/O2TOY9/mIRADOR8/F4DAB2PUGH1/I5GOOEY3V/T4XI2MALTHA/14N/6GUM3OWN/7PEW2DOE/9EF1DOR/2KUNA1J1BEVELS/3TURRETs2S2/7A4T2/7N7/7S7 EEEIILZ/ 336/298 0 lex NWL20;
+./bin/magpie sim -threads 7 -numplays 5 -scond 99 -it 10000 -pfreq 100 -cfreq 500 -plies 5
 ```
 
 Run an inference:
 
 ```
- ./bin/magpie go infer rack HUJA pindex 0 score 20 exch 0 numplays 20 info 1000000 threads 4 lex OSPS44
+ ./bin/magpie infer 0 HUJA 20 -numplays 20 -pfreq 1000000 -threads 4 -lex OSPS44
```

Comment:
player indices are now 1 and 2, not 0 and 1

Reply:
Thanks for catching! Just fixed.